build.gradle

```diff
@@ -126,12 +137,13 @@ jacocoTestReport {
 tasks.test.finalizedBy    tasks.jacocoTestReport
 tasks.sonarqube.dependsOn tasks.jacocoTestReport
 
-def jar = rootProject.name + '-' + version + ".jar"
+ext.bootJarPath    = relativePath(tasks.bootJar.outputs.files.singleFile)
+ext.gitShortCommit = 'git rev-parse --short=8 HEAD'.execute().text.trim()
+ext.ociImageName   = "local/${rootProject.name}"
 
-docker {
-	name 'docker.io/iexechub/' + rootProject.name + ':dev'
-	dockerfile file('docker/Dockerfile-local')
-	files('build/libs/' + jar)
-	buildArgs([JAR_PATH: jar ])
+task buildImage(type: Exec) {
+    group 'Build'
+    description 'Builds an OCI image from a Dockerfile.'
+    commandLine("sh", "-c", "docker build -f docker/Dockerfile-local --build-arg spring_boot_jar=$bootJarPath -t $ociImageName:$gitShortCommit ."
```

Comment:
Any reason for this style instead of using palantir plugin?

Reply:
It was just easier and quicker for me to go this route.
And with the discussions coming on finding another container runtime technology in the future, I find it more maintainable to use this implementation.
Plus, it allows to quickly copy paste the command in a terminal if needed.